plugins/module_utils/network/sonic/config/mclag/mclag.py

```diff
@@ -113,6 +152,27 @@ def execute_module(self):
         if result['changed']:
             result['after'] = changed_mclag_facts
 
+        new_config = changed_mclag_facts
+        old_config = existing_mclag_facts
+        if self._module.check_mode:
+            result.pop('after', None)
+            for command in commands:
+                self.transform_config_for_diff_check(command)
+            self.transform_config_for_diff_check(existing_mclag_facts)
+            new_config = get_new_config(commands, existing_mclag_facts,
+                                        TEST_KEYS_generate_config)
+            d_id = existing_mclag_facts.get('domain_id', None)
```

Comment:
if existing config is empty/not found this throws an error `"'list' object has no attribute 'get'"`

Reply:
existing_mclag_facts is dict type.

Reply:
it is supposed to be, I think it may be the edge case of `get_mclag_facts` returning an empty list if it does not find facts

Reply:
Fixed. Thanks for the findings!